cmd/storagenode/main.go

```diff
@@ -15,9 +16,11 @@ import (
 	"sort"
 	"text/tabwriter"
 
+	"github.com/Masterminds/semver"
 	"github.com/fatih/color"
 	"github.com/gogo/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
```

Comment:
The package and it's dependencies need to be reviewed.

Reply:
Also one of the dependencies has an issue reported with McAfee anti-virus https://github.com/inconshreveable/go-update/issues/37